src/main/java/io/phdata/streamliner/schemadefiner/configbuilder/ScriptCommand.java

```diff
@@ -7,13 +7,13 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 import java.util.stream.Collectors;
 
 public class ScriptCommand {
   private static final Logger log = LoggerFactory.getLogger(ScriptCommand.class);
+  private static Map<String, Map<String, List>> commonTables;
```

Comment:
Let us restructure this class to eliminate the requirement for static state. There are two reasons:

1. static state tends to create more static state
2. using the cost in a non-static context, say for a web service, becomes impossible

Reply:
Removed all static keywords. Now all are instance method